### PR TITLE
Adding buildConfig to templates

### PR DESCRIPTION
**Pull request by joejstuart**

### CODE REVIEW

File: config/Dockerfiles/grafana/Dockerfile [hunk 1]
```diff
@@ -1,22 +1,19 @@
 FROM centos:7
-ENV GRAFANA_VERSION=5.0.0
 
-LABEL name="Grafana" \
-      version=$GRAFANA_VERSION
+ENV GRAFANA_INSTALL grafana-5.0.0-1.x86_64.rpm
```
Comment (johnbieren) [inline config/Dockerfiles/grafana/Dockerfile]: you could do one ENV then just keep pasting them *i think* but just a nit pick

File: config/s2i/grafana/grafana-buildconfig-template.yaml [hunk 2]
```diff
@@ -17,13 +17,50 @@ parameters:
 - name: STORAGE_SIZE
   value: 5Gi
   description: The size of the persistent storage for Grafana
+- name: REPO_URL
+  value: https://github.com/joejstuart/ci-pipeline.git
```
Comment (johnbieren) [inline config/s2i/grafana/grafana-buildconfig-template.yaml]: CentOS-PaaS-SIG, not joejstuart, right?

File: config/s2i/influxdb/influxdb-buildconfig-template.yaml [hunk 3]
```diff
@@ -17,16 +17,50 @@ parameters:
 - name: STORAGE_SIZE
   value: 10Gi
   description: The size of the persistent storage for Influxdb
+- name: REPO_URL
+  value: https://github.com/joejstuart/ci-pipeline.git
```
Comment (johnbieren) [inline config/s2i/influxdb/influxdb-buildconfig-template.yaml]: same as above

File: config/s2i/telegraf/telegraf-buildconfig-template.yaml [hunk 4]
```diff
@@ -31,16 +31,53 @@ parameters:
 - name: JENKINS_ENDPOINT_TYPE
   value: metrics?pretty=true
   description:
+- name: IMAGE_STREAM_TAG
+  value: telegraf:latest
+  description: The telegraf container image
+- name: REPO_URL
+  value: https://github.com/joejstuart/ci-pipeline.git
```
Comment (johnbieren) [inline config/s2i/telegraf/telegraf-buildconfig-template.yaml]: same

File: config/s2i/grafana/grafana-buildconfig-template.yaml (hunk 2 above)
Comment (joejstuart) [reply config/s2i/grafana/grafana-buildconfig-template.yaml]: yes, good catch.

File: config/Dockerfiles/grafana/Dockerfile (hunk 1 above)
Comment (joejstuart) [reply config/Dockerfiles/grafana/Dockerfile]: Looks like you can. Testing it now.

**Pull request merged**